### PR TITLE
chore: Add snyk monitoring for master branch

### PR DESCRIPTION
**Pull request by kenoir**
This PR reliably integrates the repository with the snyk GitHub action which will scan your code’s dependencies and alert you if vulnerabilities are found. This PR has only been raised on repos that have already been tested to make sure scanning will work out of the box. ‘reliably integrated’ means that this action compares the hash of the last commit on main to the one that snyk has, and makes sure that they match. If you think that this repository doesn’t belong to your team, please mark your team as something other than an Admin for this repo before closing the PR, or its highly likely further PRs will be raised.

### CODE REVIEW

File: .github/workflows/snyk.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  workflow_dispatch:
+
+jobs:
+  security:
+    uses: guardian/.github/.github/workflows/sbt-node-snyk.yml@main
+    with:
+      DEBUG: true
+# This org name will probably need to change based on your team
+      ORG: guardian-devtools
+      SKIP_NODE: false
```
Comment (JamieB-gu) [inline .github/workflows/snyk.yml:20]: Do we want to set `SKIP_SBT` to `true`?

File: .github/workflows/snyk.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      - master
+  workflow_dispatch:
+
+jobs:
+  security:
+    uses: guardian/.github/.github/workflows/sbt-node-snyk.yml@main
+    with:
+      DEBUG: true
+# This org name will probably need to change based on your team
+      ORG: guardian-devtools
```
Comment (JamieB-gu) [inline .github/workflows/snyk.yml:19]: Does this need to change?

**Review by JamieB-gu — COMMENTED** [review]
What should we do about the existing Snyk integration for this repo? Will we end up reporting on this repo twice?

**Comment by kenoir** [comment]
Follows https://github.com/guardian/mobile-apps-article-templates/pull/1574

**Comment by kenoir** [comment]
> What should we do about the existing Snyk integration for this repo? Will we end up reporting on this repo twice?

When this is merged you can remove the other integrations.

**Review by JamieB-gu — APPROVED** [review]

**Pull request merged**